content/en/modules/nibabel/index.md

```diff
@@ -0,0 +1,164 @@
+---
+type: "modules" # DON'T TOUCH THIS ! :)
+
+# Title of your project (we like creative title)
+title: "Neuroimaging data and file structures in Python"
+
+# Your project GitHub repository URL
+github_repo:
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [python, neuroimaging, data analysis]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "Learning the basics of neuroimaging file format with Nibabel."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "reggie.png"
+---
+<!-- This is an html comment and this won't appear in the rendered page. You are now editing the "content" area, the core of your description. Everything that you can do in markdown is allowed below. We added a couple of comments to guide your through documenting your progress. -->
+
+## Information
+
+The estimated time to complete this training module is 1h30m for the basic content and 1h for the optional advanced content.
+
+ - **Basic** - Be able to load and save different types of neuroimaging files in Nibabel. Know how to work with volumetric data.
+ - **Advanced** - Understand the difference between array and array proxy images.
+ - **Optional** - Acquire a passing familiarity with the structures of surface images, CIFTI-2 files and tractogram.
+
+The prerequisites to take this module are:
+ * the [introduction to the terminal](/modules/introduction_to_terminal) module.
+ * the [DataLad](/modules/datalad) module.
+
```

Comment:
do you think we should add a python module as a prerequisite?

Reply:
yes let's add python